### PR TITLE
My Gardens Page

### PR DESCRIPTION
**Pull request by kwell755**
Screenshot 2020-07-07 at 8 23 46 PM
Screenshot 2020-07-07 at 8 22 20 PM

This is the My Gardens Page that displays all gardens that the user is apart of, either being a normal member or a admin.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,5 +9,12 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.2",
     "typescript": "^3.8.3"
+  },
+  "dependencies": {
```
Comment (codyjrivera) [inline package.json:13]: I wonder whether these were added automatically. If so, then maybe we do need to keep these in.

File: src/main/webapp/src/app/my_gardens_page/my-gardens.component.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import {Component, OnInit} from '@angular/core';
+import {OwlOptions} from 'ngx-owl-carousel-o';
+
+@Component({
+  selector: 'MyGardens',
+  templateUrl: './my-gardens.component.html',
+  styleUrls: ['./my-gardens.component.css'],
+})
+export class MyGardensComponent implements OnInit {
+  /* Option to display a certian number of slides based off of browser window size along with
```
Comment (codyjrivera) [inline src/main/webapp/src/app/my_gardens_page/my-gardens.component.ts:10]: In addition to the spelling nit @samouri pointed out, another nit that applies to both these PRs is the comment delimiting.

Use
```javascript
/**
 * Line 1 of the comment
 * Line 2 of the comment
 * Line 3 of the comment
 */
```

File: package.json (hunk 1 above)
Comment (kwell755) [reply package.json:13]: Yeah they were all added automatically. I'll keep them in going forward

File: src/main/webapp/src/app/app.module.ts [hunk 3]
```diff
@@ -18,24 +18,32 @@ import {NgModule} from '@angular/core';
 import {AppRoutingModule} from './app-routing.module';
 import {MaterialComponents} from './common/material-components';
 import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+import {MatCardModule} from '@angular/material/card';
+import {CarouselModule} from 'ngx-owl-carousel-o';
```
Comment (stephypy) [inline src/main/webapp/src/app/app.module.ts:22]: I have a question about the carousel: Is this carousel the only one available for angular or does angular have its own carousel?

File: src/main/webapp/src/app/my_gardens_page/my-gardens.component.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
+.avatar {
+  background-size: cover;
+  height: 50px;
+  padding-top: 5px;
+}
+.avatar-image {
+  width: 50px;
+}
+
+.button {
```
Comment (stephypy) [inline src/main/webapp/src/app/my_gardens_page/my-gardens.component.css:10]: Since it seems like we want to keep the same style for buttons, we should move this styling to the common folder. I  would say these styling changes should be made in a different PR so it can be changed in all files evenly.

File: src/main/webapp/src/app/my_gardens_page/my-gardens.component.css [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  padding: 0.5em 3em;
+  text-decoration: none;
+  border: none;
+  box-sizing: border-box;
+  box-shadow: inset 0 -0.6em 0 -0.35em #003353;
+  background-color: #005c80;
+  color: white;
+}
+.cards {
+  max-width: 400px;
```
Comment (stephypy) [inline src/main/webapp/src/app/my_gardens_page/my-gardens.component.css:23]: nit: make sure all css functions are separated by one line.
@codyjrivera can you please comment the linting command here? I also forget about it

Comment (kwell755) [reply src/main/webapp/src/app/my_gardens_page/my-gardens.component.css:23]: Its ok, I know the linting command. When I ran it, it didn't change that.

File: src/main/webapp/src/app/app.module.ts (hunk 3 above)
Comment (kwell755) [reply src/main/webapp/src/app/app.module.ts:22]: In my research I didn't see a material carousel component. You either had to hard code it yourself,use ng bootstrap, or use a different package. https://material.angular.io/components/categories

File: src/main/webapp/src/app/my_gardens_page/my-gardens.component.css (hunk 5 above)
Comment (codyjrivera) [reply src/main/webapp/src/app/my_gardens_page/my-gardens.component.css:23]: @stephypy `make format` and `make lint` in the top directory

File: src/main/webapp/src/app/my_gardens_page/my-gardens.component.ts (hunk 2 above)
Comment (kwell755) [reply src/main/webapp/src/app/my_gardens_page/my-gardens.component.ts]: Done

File: src/main/webapp/src/app/common/growpod-page-styles.css [hunk 6]
```diff
@@ -9,3 +9,15 @@
 .text-center {
   text-align: center;
 }
+.button {
+  display: flex;
+  margin-left: auto;
```
Comment (samouri) [inline src/main/webapp/src/app/common/growpod-page-styles.css:14]: Can you create the same effect via `margin: 0 auto;`. I think that is the more common/recommended way to write it.

Comment (kwell755) [reply src/main/webapp/src/app/common/growpod-page-styles.css]: Done
